packages/ckeditor5-dev-release-tools/lib/tasks/cleanuppackages.js

```diff
@@ -0,0 +1,173 @@
+/**
+ * @license Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const fs = require( 'fs-extra' );
+const upath = require( 'upath' );
+const { globSync } = require( 'glob' );
+const { logger } = require( '@ckeditor/ckeditor5-dev-utils' );
+
+/**
+ * The purpose of the script is to clean all packages prepared for the release. The cleaning consists of two stages:
+ *
+ * - Remove unnecessary files and directories from the package directory.
```

Comment:
It is worth mentioning that it filters out files/directories not included in the `files` property in pkg.json.

Reply:
Added more details.